### ISSUE TITLE
Clinical Kernel

### ISSUE THREAD

**Issue by annette987** [imbs-hl/survivalsvm#5]
I have been using  'survivalsvm' but have just learnt from this paper https://www.elen.ucl.ac.be/Proceedings/esann/esannpdf/es2010-16.pdf  about the idea of a clinical kernel for survivalsvm. I was wondering if the clinical kernel has been implemented in R and if so how would I access it. Thanks.

**Comment by mnwright** [comment]
It's not yet implemented in the package. Do you know of any other implementation?

**Comment by annette987** [comment]
Yes, there is a Python implementation here:  [https://github.com/sebp/scikit-survival](https://github.com/sebp/scikit-survival)
See scikit-survival/sksurv/kernels/clinical.py

**Comment by mnwright** [comment]
Thanks, that looks doable. 
@fouodo What do you think?

**Comment by fouodo** [comment]
I think we can integrate that to the package. I will add the task into my toDo list.

Regards,
Cesaire

**Comment by annette987** [comment]
Thank you very much.

**Comment by annette987** [comment]
Just wondering if you have had any time to start on this task yet.

**Comment by fouodo** [comment]
Hi Annette,

sorry, not yet. But I will try to do it in the next day. 
Thank you to remember me about that.

Best regards,
Cesaire

**Comment by annette987** [comment]
Hi Cesaire,
I really appreciate you doing this. Have you had a chance to make any progress on it yet?
Thanks,
Annette
